web/src/lib/components/shared-components/search-bar/search-text-section.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+  export let fileName: string | undefined;
+  export let context: string | undefined;
+
+  enum TextSearchOptions {
+    Context = 'context',
+    FileName = 'file-name',
+  }
```

Comment:
The code on this page will have to be refactored a lot if we ever have more than two radio buttons. IMO we shouldn't anyway, but just something to keep in mind.